flask_app/controllers/projects.py

```diff
@@ -26,6 +27,7 @@ def projects():
     data ={
         'id': session['user_id']
     }
+    # should project page return a project by id with its tasks?
     return render_template("project.html",user=User.get_by_id(data), new_system = Project.get_all_projects())
```

Comment:
I think the dashboard should iterate through a user's projects. So this could be a get_user_by_id_with_projects

Reply:
Hmmm I think I know what you mean! Would that be along the line of get_all_projects? Except without specifying the user_id or would we want only the specific user to be able to see the projects?

Reply:
The latter of what you said. I think for dashboard, we want to grab the user (by id) with all projects associated with them.

Reply:
So it would end up being get_all_projects_<id>. This means we would have to add a new route! I might be able to get it done but I will lt you know if I wont

Reply:
I think it would be another class method , not another controller route per se. Since a user can have many projects, and dashboard displays all of one user's projects, we can just grab that user and add to the user's class. So actually, this may be something for @Teknicl to work on since he was working on user. My bad, @aacole9, didn't mean to confuse you.

I added a line to user.py to show what I'm talking about. So I think we are looking at method that grabs a user with all their projects, and route in users.py to render dashboard. I don't think this is a project.py/projects.py issue.

![Screenshot 2023-01-30 at 9 20 23 PM](https://user-images.githubusercontent.com/107015116/215643823-522f598b-0215-42b0-8fe6-ca601f2602bb.png)

Reply:
I agree. I think it should be all the projects designated to that user. We don't have a comment section for one user to comment on another users project. If that was the case then I would say dashboard would have to display all.